pyproject.toml

```
@@ -41,6 +41,7 @@ dev-dependencies = [
     "time-machine==2.9.0",
     "nox==2023.4.22",
     "dirty-equals>=0.6.0",
+    "azure-identity >=1.14.1",
```

We released 1.15.0 which contains the `get_bearer_token_provider` auth helper

```suggestion
    "azure-identity >=1.15.0",
```